chia/cmds/plotnft_funcs.py

```diff
@@ -120,7 +119,10 @@ async def create(args: dict, wallet_client: WalletRpcClient, fingerprint: int) -
                     print(f"Do chia wallet get_transaction -f {fingerprint} -tx 0x{tx_record.name} to get status")
                     return None
         except Exception as e:
-            print(f"Error creating plot NFT: {e}")
+            print(
+                f"Error creating plot NFT: {e}\n  Please ensure both farmer and wallet are running:"
+                f"    chia start -r wallet farmer-only"
```

Comment:
I have a pending PR out to change wallet to only start the wallet and not the node.  I think just recommending `chia start -r farmer` here is better because it will start everything the farmer needs.

Reply:
That's a good idea, but doesn't this code use the wallet RPC API above at this line?

`tx = await wallet_client.get_transaction(str(1), tx_record.name)`

Reply:
It does but `chia start farmer` starts the wallet too https://github.com/Chia-Network/chia-blockchain/blob/00613bc259584c7f9c06c3f7aca2a52e1da72b2c/chia/util/service_groups.py#L7

Reply:
Good idea